QRScanner/QRScannerView.swift

```diff
@@ -9,12 +9,20 @@
 import UIKit
 import AVFoundation
 
+// MARK: - QRScannerViewDelegate
 public protocol QRScannerViewDelegate: AnyObject {
-    func failure(_ error: QRScannerError)
-    func success(_ code: String)
-    func didChangeTorchActive(isOn: Bool)
+    // Required
+    func qrScannerView(_ qrScannerView: QRScannerView, didFailure error: QRScannerError)
+    func qrScannerView(_ qrScannerView: QRScannerView, didSuccess code: String)
+    // Optional
+    func qrScannerView(_ qrScannerView: QRScannerView, didChangeTorchActive isOn: Bool)
 }
 
+public extension QRScannerViewDelegate {
+    func qrScannerView(_ qrScannerView: QRScannerView, didChangeTorchActive isOn: Bool) {}
```

Comment:
Add default implementation

Of course, in another way we can add `optional func` at protocol declaration.
But it is so confused 😨 

```swift
@objc public protocl QRScannerViewDelegate: AnyObject {
    func qrScannerView(...)
    func qrScannerView(...)
    @objc optional func qrScannerView(...)
```

But this causes error because `QRScannerError` enum is also needed `@objc`

```swift
@objc public enum QRScannerError {
    case ...

    @objc public enum DeviceError {
        case ...
    }
}
```

`QRScannerError.DeviceError` is also needed to append `@objc`...

So I avoid to append `@objc` attribute and add default implementation.